File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/PopulatingData/BindingDataCollectionPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.ListView.PopulatingData
{
    public partial class BindingDataCollectionPage : UserControl
    {
        public BindingDataCollectionPage()
        {
            InitializeComponent();
            
            // Set DataSources for the ListView as Customer objects collection.
            this.mobjListView.DataSource = Gizmox.WebGUI.Forms.CompanionKit.Controls.Util.CustomerData.GetCustomers();
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/PopulatingData/BindingDataCollectionPage.vb
Namespace CompanionKit.Controls.ListView.PopulatingData

	Public Class BindingDataCollectionPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

            '  Set DataSources for the ListView as Customer objects collection.
            Me.mobjListView.DataSource = Global.CompanionKit.Controls.Utils.CustomerData.GetCustomers

		End Sub
	End Class

End Namespace